lighthouse-cli/printer.js

```diff
@@ -90,22 +90,16 @@ function writeFile(filePath, output, outputMode) {
  * @param {!LH.Results} results
  * @param {string} mode
  * @param {string} path
- * @return {!Promise<!LH.Results>}
+ * @return {!Promise<void>}
  */
 function write(results, mode, path) {
-  return new Promise((resolve, reject) => {
-    const outputPath = checkOutputPath(path);
-    const output = createOutput(results, mode);
+  const outputPath = checkOutputPath(path);
```

Comment:
this looks just like cleanup no functional change right?

Reply:
yup. though i remember some bizarre rejection problems were, i think, why we complicated this method. we'll see.